dangerfile.js

```diff
@@ -0,0 +1,27 @@
+import { danger, fail, markdown, message, peril, schedule, warn } from 'danger'
+
+// Enforce our Gemfile update policy
+if (danger.git.modified_files.includes("Gemfile.lock") &&
+    danger.github.pr.user.type !== "Bot") {
+    /*
+     * our default template now includes --conservative, so we have to
+     * strip that out. Oops.
+     */
+    body = danger.github.pr.body
+    body = body.replace("I have used `--conservative` to do it", "")
+
+    if (danger.git.modified_files.includes("Gemfile")) {
+        message("PR updates Gemfile.lock, but it also updates Gemfile, so that" +
```

Comment:
I wonder if extracting the message contents into a variable would improve the readability some. Might be overkill.

Reply:
I think it'd get confusing as we add more rules? I guess we could make a constants import which we then import into this, but... I'm lazy. :)